akka-cluster-tools/src/main/scala/akka/cluster/client/ClusterClient.scala

```diff
@@ -174,12 +189,17 @@ class ClusterClient(settings: ClusterClientSettings) extends Actor with Stash wi
 
   require(initialContacts.nonEmpty, "initialContacts must be defined")
 
+  val acceptableHeartbeatPauseNanos: Long = acceptableHeartbeatPause.toNanos
```

Comment:
I was thinking that it was so simple that I don't needed any fancy stuff, but I guess you are right, I'll give that a try